addon/services/intl.js

```diff
@@ -52,6 +52,10 @@ const IntlService = Service.extend(Evented, {
   formatTime: formatterProxy('time'),
   formatDate: formatterProxy('date'),
 
+  t(...args) {
+    return this.formatMessage(this.findTranslationByKey(...args));
```

Comment:
formatMessage takes 3 arguments: the translation, the value hash, and the format hash